clang/test/Interpreter/out-of-process.cpp

```diff
@@ -0,0 +1,61 @@
+// REQUIRES: host-supports-jit, x86_64-linux
```

Comment:
I believe this test copies some content from other tests. Would it make sense to add an extra `RUN` line to the tests themselves?

Reply:
Since only ELF and Linux are supported right now, I thought that adding extra `RUN` lines to the others tests would cause unnecessary failures in COFF (Windows) and MachO (Mac). Would there be a way to conditionally execute `RUN` lines based on platform?